mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -915,6 +917,22 @@ class GeneralizedLinearRegressionSummary private[regression] (
   /** Number of instances in DataFrame predictions. */
   private[regression] lazy val numInstances: Long = predictions.count()
 
+
+  /**
+   * Name of features. If the name cannot be retrieved from attributes,
+   * set default names to "V1", "V2", and so on.
+   */
+  @Since("2.2.0")
+  lazy val featureName: Array[String] = {
+    val featureAttrs = AttributeGroup.fromStructField(
+      dataset.schema(model.getFeaturesCol)).attributes
+    if (featureAttrs == None) {
+      Array.tabulate[String](origModel.numFeatures)((x: Int) => ("V" + (x + 1)))
```

Comment:
@felixcheung The feature names were not available prior to this PR, right? One other place I see that does similar summary is the `GeneralizedLinearRegressionWrapper` for R. Do you think we should consolidate the two, e.g., update the `Wrapper` to use the summary table directly?